### PR TITLE
SEP6-11 Search for movies & people

### PR DESCRIPTION
**Pull request by RealFoXsTeRDOTA**

### CODE REVIEW

File: src/MovieManagement/Components/Search.razor.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using MovieManagement.ViewModels.Search;
```
Comment (HolaSombreros) [inline src/MovieManagement/Components/Search.razor.cs:1]: this should be in global usings, no?

File: src/MovieManagement/Pages/SearchResults/SearchResults.razor.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿using MovieManagement.ViewModels.Search;
```
Comment (HolaSombreros) [inline src/MovieManagement/Pages/SearchResults/SearchResults.razor.cs:1]: global usings?

File: src/MovieManagement/appsettings.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
   },
   "AzureFunctionsConfig": {
     "AzureFunctionUri": "https://moviemanagementfunctions.azurewebsites.net/api",
-    "HostKey": "code=",
+    "HostKey": "code=moviemanagementauthBz_wcc9CwsHGuSIAnTIgTq",
```
Comment (HolaSombreros) [inline src/MovieManagement/appsettings.json]: delete this

File: src/MovieManagement/Shared/Sidebar.razor.cs [hunk 4]
```diff
@@ -1,24 +1,26 @@
-﻿namespace MovieManagement.Shared;
+﻿using MovieManagement.ViewModels.Search;
```
Comment (HolaSombreros) [inline src/MovieManagement/Shared/Sidebar.razor.cs:1]: global usings

**Review by HolaSombreros — APPROVED** [review]

File: src/MovieManagement/Components/SearchResults/SearchMovieCard.razor [hunk 5]
```diff
@@ -0,0 +1,18 @@
+﻿@inject NavigationManager NavigationManager
+
+<img src="@Movie.PosterUrl" alt="Image not found" @onclick="@(() => ShowMovieDetails(Movie.Id))" />
+<h4>@Movie.Title</h4>
+@if (Movie.ReleaseDate != null)
+{
+    <p>Release date: @Movie.ReleaseDate</p>
+}
+
+@code {
```
Comment (vladdirlea2002) [inline src/MovieManagement/Components/SearchResults/SearchMovieCard.razor:10]: inside code

File: src/MovieManagement/Components/SearchResults/SearchPersonCard.razor [hunk 6]
```diff
@@ -0,0 +1,15 @@
+﻿@inject NavigationManager NavigationManager
+
+<img src="@Person.ImageUrl" alt="Image not found" @onclick="(() => ShowPersonDetails(Person.Id))" />
+<h4>@Person.Name</h4>
+<p>Known for: @Person.KnownFor</p>
+
+@code {
```
Comment (vladdirlea2002) [inline src/MovieManagement/Components/SearchResults/SearchPersonCard.razor:7]: inside code

File: src/MovieManagement/GlobalUsings.cs [hunk 7]
```diff
@@ -26,4 +26,6 @@
 global using System.Diagnostics;
 global using System.Security.Claims;
 global using System.Text.Json;
-global using System.Text.Json.Serialization;
```
Comment (vladdirlea2002) [inline src/MovieManagement/GlobalUsings.cs:29]: I struggled to order them alphabetically. Please do so as well so we don't end up with 100 duplicates

File: src/MovieManagement/Shared/MainLayout.razor.css [hunk 8]
```diff
@@ -42,6 +42,6 @@ main {
     }
 }
 
-::-webkit-scrollbar {
+/*::-webkit-scrollbar {
```
Comment (vladdirlea2002) [inline src/MovieManagement/Shared/MainLayout.razor.css:45]: Make it visible and display it if you want it on buddy. Don't comment it and let it be displayed without css -.-'

**Review by vladdirlea2002 — CHANGES_REQUESTED** [review]

**Review by stefaniatomuta — APPROVED** [review]

**Review by vladdirlea2002 — APPROVED** [review]

**Pull request merged**